pinot-plugins/pinot-input-format/pinot-avro-base/src/test/java/org/apache/pinot/plugin/inputformat/avro/AvroRecordExtractorTest.java

```diff
@@ -73,8 +73,12 @@ protected void createInputFile()
             new Field("firstName", createUnion(Lists.newArrayList(create(Type.STRING), create(Type.NULL))), null, null),
             new Field("lastName", createUnion(Lists.newArrayList(create(Type.STRING), create(Type.NULL))), null, null),
             new Field("bids", createUnion(Lists.newArrayList(createArray(create(Type.INT)), create(Type.NULL))), null,
-                null), new Field("campaignInfo", create(Type.STRING), null, null),
-            new Field("cost", create(Type.DOUBLE), null, null), new Field("timestamp", create(Type.LONG), null, null));
+                null),
+            new Field("campaignInfo", create(Type.STRING), null, null),
+            new Field("cost", create(Type.DOUBLE), null, null),
+            new Field("timestamp", create(Type.LONG), null, null),
+            new Field("xarray", createArray(create(Type.STRING))),
+            new Field("xmap", createMap(create(Type.STRING))));
```

Comment:
Suggest adding all supported types (BOOLEAN, ENUM, FIXED, RECORD)

Reply:
This file is shared by Thrift, Protobuf, ORC, CSV, etc. and I am seeing some differences in handling of boolean (Thrift vs others), fixed (custom code needed for setting Fixed values in Avro, vs binary in others), and enum (custom code needed for Avro) between different file formats, so probably a better idea to do this in a separate PR of its own.